test/issues/issues-test.js

```diff
@@ -180,4 +180,21 @@ describe("issues", function () {
             assert(firstFake !== secondFake);
         });
     });
+
+    describe("#1372 - sandbox.resetHistory", function () {
+        it("should reset spies", function () {
+            var spy = this.sandbox.spy();
+
+            spy();
+            assert.equals(spy.callCount, 1);
+
+            spy();
+            assert.equals(spy.callCount, 2);
+
+            this.sandbox.resetHistory();
```

Comment:
I'm assuming there are already related tests for stubs. It wouldn't hurt to have a test with both since you're potentially affecting both.